Plugins/ExternalTexture/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+set(SOURCES
+    "Include/Shared/Babylon/Plugins/ExternalTexture/ExternalTexture.h"
+    "Source/ExternalTexture_${GRAPHICS_API}.cpp"
+    "Source/ExternalTexture.cpp")
+
+add_library(ExternalTexture ${SOURCES})
+warnings_as_errors(ExternalTexture)
+
+target_include_directories(ExternalTexture
+    PUBLIC "Include/Shared"
+    PRIVATE "Source")
+
+if(WIN32 AND NOT WINDOWS_STORE)
+    target_link_to_dependencies(ExternalTexture
+        PRIVATE "shlwapi.lib")
+elseif(ANDROID)
+    target_link_to_dependencies(ExternalTexture
+        PRIVATE android
+        PRIVATE AndroidExtensions)
+endif()
+
+target_link_to_dependencies(ExternalTexture
+    PRIVATE napi_extensions
+    PRIVATE JsRuntime
+    PRIVATE JsRuntimeInternal
+    PRIVATE GraphicsDevice
```

Comment:
Should these be public dependencies? Your header depends directly on JsRuntime and GraphicsDevice concepts, at least.

Reply:
Moved JsRuntiem and GraphicsDevices to be public dependencies.